src/DetailsView/components/assessment-instance-selected-button.scss

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/colors.scss';
+
+:global(.test-step-view) {
```

Comment:
This shouldn't be necessary; the class will only apply to where it is specifically imported.

Reply:
Removed and validated